src/lib/locale.ts

```diff
@@ -0,0 +1,9 @@
+/**
+ * Splits locale by hyphen
+ *
+ * @internal
+ */
+export function splitLocale (locale : string) : string[]
+{
+	return locale.split("-");
```

Comment:
We should refactor this, so that we a) validate the input and b) return `{language: string, country: string|null}`